### DISCUSSION TITLE
Most Current USB Drivers?

### DISCUSSION THREAD

**Discussion by CRhodes-123** [bitcraze/discussions#308] [Q&A]
Hello everyone,
I'm Collin Rhodes, a University student doing research with crazyflies. 

I recently reinstalled the Bitcraze VM to make sure everything I have is the newest version, and everything seems to be working properly again which is great except for some issues I have had with connecting to the crazyflie. I have set the filters in oracle correctly so there should be no issue there, but when I go to the crazyflie client and select for it to search, in the terminal I get a warning about not being able to find a crazyradio dongle and a separate warning about usb driver, which I have included a photo of below. 
![image](https://user-images.githubusercontent.com/103008098/194801317-29dad8b6-6196-4dae-a6d1-02890b8e29f8.png)

I looked at the documentation for usb drivers, and found that I am supposed to use zadig to apply the libusb-win32 (v1.2.6.0) driver to the crazyradio usb dongle, but when I go to do that I get an error message saying the driver installation failed. 

Whether the issue is with the usb filters or otherwise, any help would be appreciated, and I would be more than happy to follow up with more details if any elaborations or clarifications need to be made

Thanks in advance,
Collin Rhodes

**Comment by ataffanel** [comment]
There is no driver needed on the Linux side, you just need to make sure the USB device access right are set in a udev rules file but this is already done in the vm.

In order for the VM to have access to the radio dongle it needs to be passed to it using either USB filter or the Devices/usb menu. Make sure that the Crazyradio is marked in the Devices/usb menu.
